engine/src/main/java/org/terasology/engine/subsystem/headless/renderer/HeadlessWorldRenderer.java

```diff
@@ -127,9 +119,18 @@ public void update(float delta) {
     }
 
     @Override
+    public void increaseTrianglesCount(int increase) {
+        // we are not going to count triangles in headless
+    }
+
+    @Override
+    public void increaseNotReadyChunkCount(int increase) {
+        // we are not going to count not ready chunks in headless
+    }
```

Comment:
These two new methods above are a bit of a hack to maintain existing functionality. They are not horrible and they could probably stay for a while. But I'm thinking we might want to have some kind of StatisticsManager registered in the main context holding all sorts of statistics like these. That would be a more appropriate place I feel. Or we could keep those statistics here and just make something that wraps them a bit better than what we have now.